src/datasources/perf-ds/datasource.ts

```diff
@@ -341,6 +343,29 @@ export class OpenNMSDatasource {
     return Promise.resolve([]);
   }
 
+  metricFindNodesLocation(searchLimit) {
+    return this.doOpenNMSRequest({
+      url: '/rest/nodes',
```

Comment:
There is a `monitoringLocations` REST API which should give you the requested information without provoking a full table scan on the nodes table.

Reply:
I was wondering about that. Looked though the documentation and couldn't find one. I'll fix that. Thanks!

Reply:
switched rest call to use monitoringLocations instead